index.html

```diff
@@ -39,11 +39,133 @@ <h1>Nsong Fitz Gerald</h1>
                 <div class="button">
                     <button id="introbutton">Start collaboration</button>
                 </div>
+            </div> 
+        </section>
+        <section>
```

Comment:
:red_circle: :red_circle: [The project requiremen](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)t states that you should combine both Grid and Flexbox to lay out elements in this section . You used only flexbox. Kindly implement one of the project cards with CSS Grid  :arrow_down: 

![requir](https://user-images.githubusercontent.com/26736582/205437463-d7720e99-227b-4783-9550-481daa34a89c.png)